### PR TITLE
[WFSSL-79] Create a CONTRIBUTING.md file for new contributors

### PR DESCRIPTION
**Pull request by Ashpan**
https://issues.redhat.com/browse/WFSSL-79

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+Then, in your terminal, paste the following command:
+```bash
+git clone [URL]
+```
+Be sure to replace [URL] with the URL that you copied.
+
+Now you have the repository on your computer!
+
+## Issues
+The Wildfly OpenSSL project uses JIRA to manage issues. All issues can be found [here](https://issues.redhat.com/projects/WFSSL/issues).
```
[the remark this answers is not in the document]
Comment (fjuma) [reply CONTRIBUTING.md:35]: The project used to use GitHub Issues but we switched to WFSSL recently.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+Contributing to Wildfly OpenSSL
```
[the remark this answers is not in the document]
Comment (Ashpan) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+```bash
+mvn clean install -DskipTests=true
+```
+
+To run only a specific test, use:
+
+```bash
+mvn clean install -Dtest=TestClassName
+```
+For more information, including details on how Wildfly OpenSSL is integrated in Wildfly Elytron, WildFly Core and WildFly, check out our [developer guide](https://wildfly-security.github.io/wildfly-elytron/getting-started-for-developers/).
```
[the remark this answers is not in the document]
Comment (Ashpan) [reply CONTRIBUTING.md]: Fixed

**Comment by fjuma** [comment]
@Skyllarr Please review and merge when you get a chance. Thanks!

**Pull request merged**